### ISSUE TITLE
Bug of Normalisation

### ISSUE THREAD

**Issue by Linnore** [IBM/Multi-GNN#10]
https://github.com/IBM/Multi-GNN/blob/252b0252afca109d1d216c411c59ff70753b25fc/data_loading.py#L108-L127

The data normalisation in line 127 does not normalise the last edge attribute, which corresponds to "payment format" . This is a desired behaviour if no ports nor time deltas are added. However, when ports or time deltas are added, the last edge attribute would not be "payment format" anymore and required normalisation.

**Comment by AlessioBoetti** [comment]
Agree. I'd like to add to this that when using Reverse MP, inside the `create_hetero_obj` function the `edge_attr` relative to the ports are swapped for the reverse edge, but this doesn't take into account the use of timedeltas, which if used together with the ports add other two columns, so the latest two columns are no more relative to the ports but relative to the timedeltas.
